src/Common/ReflectionHelper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
```

Comment:
copyright

Reply:
??

Reply:
At the top of our files we put a copyright notice
https://github.com/aspnet/SignalR/blob/748e9928651e047d5e9e1e7020f3cd3985496b3d/src/Microsoft.AspNetCore.Http.Connections/ConnectionsRouteBuilder.cs#L1-L2